edge/docs/installation/inception_internals.md

```diff
@@ -0,0 +1,122 @@
+
+Any time a modification is made to any of the configuration files, the startedge.sh script should be run to restart the containers so they will pick up the new settings.
+### Logging Levels
+Logging levels can be changed to increase or decrease the detail of information written to the service container logs. 
+
+Logging levels are specified in the `<install root>/volume/run/var/config/vision-edge.properties` as:
+|Variable|Description|
+|-|-|
+|CONTROLLER_LOG_LEVEL|Sets the vision-edge-controller container's logging level|
+|CME_LOG_LEVEL|Sets the vision-edge-cme container's logging level|
+|DLE_LOG_LEVEL|Sets the vision-edge-dle container's logging level|
+
+The valid logging levels are:
+|Setting|Description|
+|-|-|
+|TRACE|This is the most verbose logging level, probably only useful to developers working on the product or in extreme cases where troubleshooting at DEBUG level is not providing enough detail.
+|DEBUG|Output detailed debugging log messages. This should only be used when troubleshooting the system or when detailed logging is desired when performing development or system integration activity. This level will create very large log files quickly and should not be enabled for extensive periods in production. Setting log file rotation is highly recommended to avoid running out of disk space.
+|INFO|This is the default and provides a level of information suitable for a stable system
+|WARN|This level provides information that is of special significance, either non-fatal errors or events such as ancillary services being started or stopped. It is suitable for a stable production system.
+|ERROR|This level provides information when internal errors occur that may impact the functionality of the system.
+|PANIC|This level provides information when events that would normally cause a system crash occur. The MVIE containers implement panic recovery where possible, so the system processes should restart automatically but whenever events of this severity occur, they should be addressed.
+
+### Development Mode
+Setting `DEVMODE=true` in the `<install root>/volume/run/var/config/vision-edge.properties` will start the vision-edgecontroller container in development mode. When started in this mode:
+- Swagger documentation for the controller's REST API is enabled at https://`<host name>`/swagger/index.html
+- Additional configuration info is displayed in the controller logs (DEBUG logging level must also be enabled). Since the controller gets configuration information from (in order of decreasing precedence):
+  - environment variables set in the docker run command for the controller in the startedge.sh script
+  - the vision-edge.properties file
+  - the controller.json file
+  - default values
+
+  it can be useful to see it all in one place and to see which setting has taken precedence when the container is started.
```

Comment:
Capital i at start of sentence?

Reply:
> Capital i at start of sentence?

That is intended to be a continuation of `Since the controller gets configuration from` which is why it's not capitalized.